src/unix/kqueue.c

```diff
@@ -334,6 +334,17 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
         continue;
       }
 
+#if UV__KQUEUE_EVFILT_USER
+      if (ev->filter == EVFILT_USER) {
+          w = &loop->async_io_watcher;
+          assert(fd == w->fd);
+          uv__metrics_update_idle_time(loop);
+          w->cb(loop, w, w->events);
+          nevents++;
+          continue;
```

Comment:
style nit: fix indent
```suggestion
        w = &loop->async_io_watcher;
        assert(fd == w->fd);
        uv__metrics_update_idle_time(loop);
        w->cb(loop, w, w->events);
        nevents++;
        continue;
```